applications/serial_lte_modem/Kconfig

```diff
@@ -185,7 +185,7 @@ config SLM_CMUX
 	bool "CMUX support in SLM"
 
 config SLM_CMUX_UART_BUFFER_SIZE
-	int "UART Buffer size for CMUX"
+	int "UART buffer size for CMUX"
```

Comment:
should be fixed in original commit not added then fixed in 2 commits

Reply:
Fixed. Thanks.